src/pretix/eventyay_common/templates/eventyay_common/event/index.html

```diff
@@ -0,0 +1,100 @@
+{% extends "eventyay_common/event/base.html" %}
+{% load i18n %}
+{% load eventurl %}
+{% load bootstrap3 %}
+{% load static %}
+{% load eventsignal %}
+{% block title %}{{ request.event.name }}{% endblock %}
+{% block content %}
+    <nav id="event-nav" class="header-nav">
+        <div class="navigation">
+            <div class="navigation-title">
+                <h1>
+                    {{ request.event.name }}
+                    <small>
+                        {% if request.event.has_subevents %}
+                            {% trans "Event series" %}
+                        {% else %}
+                            {{ request.event.get_date_range_display }}
+                        {% endif %}
+                    </small>
+                </h1>
+            </div>
+            {% include "eventyay_common/event/component_link.html" %}
+        </div>
+    </nav>
+
+    {% eventsignal request.event "pretix.control.signals.event_dashboard_top" request=request %}
+    {% if actions|length > 0 %}
+        <div class="panel panel-danger">
+            <div class="panel-heading">
+                <h3 class="panel-title">
+                    {% trans "Your attention is required to resolve the following issues" %}
+                </h3>
+            </div>
+            <ul class="list-group">
+                {% for action in actions %}
+                    <li class="list-group-item logentry">
+                        <p>
+                            <a href="{% url "control:event.requiredaction.discard" event=request.event.slug organizer=request.event.organizer.slug id=action.id %}"
```

Comment:
Use different quote style (`"`, `'`) when nesting.

Btw, the code to build URL looks long. You should split it to a variable. [Doc](https://docs.djangoproject.com/en/5.1/ref/templates/builtins/#url).

```django
{% url 'some-url-name' arg arg2 as the_url %}
```